packages/compass-connections/package.json

```diff
@@ -28,6 +28,13 @@
     ".": "./src/index.ts"
   },
   "types": "./dist/index.d.ts",
+  "mocha": {
+    "extension": [
+      "ts"
+    ],
+    "file": "./test/mocha-plugins.ts",
+    "recursive": true
+  },
```

Comment:
There is a `.mocharc.js` file in this package that already has mocha configuration, I would have a slight preference for us to have all the configuration in one place instead of multiple (I am also not sure if mocha will actually merge this and not just use this one instead of a shared config)